presto-common/src/main/java/com/facebook/presto/common/type/TimeZoneKey.java

```diff
@@ -140,7 +144,7 @@ public static TimeZoneKey getTimeZoneKeyForOffset(long offsetMinutes)
         }
 
         if (!(offsetMinutes >= OFFSET_TIME_ZONE_MIN && offsetMinutes <= OFFSET_TIME_ZONE_MAX)) {
-            throw new PrestoException(INVALID_FUNCTION_ARGUMENT, String.format("Invalid offset minutes %s", offsetMinutes));
+            throw new InvalidFunctionArgumentException(format("Invalid offset minutes %s", offsetMinutes));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format long: offsetMinutes (declared at line 140). This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 147. Consider formatting this data with &#39;%d&#39; instead.